agent/lib/kontena/service_pods/creator.rb

```diff
@@ -105,16 +101,13 @@ def run_hooks(service_container, type)
       # @param [String] type
       def log_hook_output(id, lines, type)
         lines.each do |chunk|
-          msg = {
-              event: 'container:log',
-              data: {
-                  id: id,
-                  time: Time.now.utc.xmlschema,
-                  type: type,
-                  data: chunk
-              }
+          data = {
+              id: id,
+              time: Time.now.utc.xmlschema,
+              type: type,
+              data: chunk
           }
-          Celluloid::Actor[:queue_worker].send_message(msg)
+          rpc_client.async.notification('/containers/log', [data])
```

Comment:
The `async` makes sense here, no need to block the `ServicePods::Creator` actor if the `RpcClient` actor is busy.